bcda/models/models.go

```diff
@@ -67,6 +151,22 @@ type ACO struct {
 	ACOBeneficiaries []*ACOBeneficiary
 }
 
+func (aco *ACO) GetBeneficiaryIDs() (beneficiaryIDs []string, err error) {
+	db := database.GetGORMDbConnection()
+	defer database.Close(db)
+
+	beneficiaryJoin := "inner join beneficiaries on acos_beneficiaries.beneficiary_id = beneficiaries.id"
+	if err = db.Table("acos_beneficiaries").Joins(beneficiaryJoin).Where("acos_beneficiaries.aco_id = ?", aco.UUID).Pluck("beneficiaries.blue_button_id", &beneficiaryIDs).Error; err != nil {
+		log.Errorf("Error retrieving ACO-beneficiaries for ACO ID %s: %s", aco.UUID.String(), err.Error())
+		return nil, err
+	} else if len(beneficiaryIDs) == 0 {
+		log.Errorf("Retrieved 0 ACO-beneficiaries for ACO ID %s", aco.UUID.String())
+		return nil, fmt.Errorf("Retrieved 0 ACO-beneficiaries for ACO ID %s", aco.UUID.String())
```

Comment:
For the error being returned, not capitalizing is the standard practice https://github.com/golang/go/wiki/CodeReviewComments#error-strings (makes sense to capitalize in the previous line for the log message though)

Reply:
will fix in incoming commit